client/src/client_error.rs

```diff
@@ -52,7 +52,6 @@ impl Into<TransportError> for ClientErrorKind {
 pub struct ClientError {
     command: Option<&'static str>,
     #[source]
-    #[error(transparent)]
```

Comment:
@t-nelson We got this error and I think removing just this is what you wanted at #8804 Is this right?

https://buildkite.com/solana-labs/solana/builds/23500#7774e986-886e-4d6d-9c3c-c1c4b4f852a3/78-123

```
error: #[error(transparent)] needs to go outside the enum or struct, not on an individual field
  --> client/src/client_error.rs:55:5
   |
55 |     #[error(transparent)]
   |     ^^^^^^^^^^^^^^^^^^^^^
 
error: #[error(transparent)] needs to go outside the enum or struct, not on an individual field
  --> client/src/client_error.rs:55:5
   |
55 |     #[error(transparent)]
   |     ^^^^^^^^^^^^^^^^^^^^^
 
    Checking solana-runtime v1.2.0 (/solana/runtime)
```